server/src/infra/ldap_handler.rs

```diff
@@ -2772,6 +3012,97 @@ mod tests {
         );
     }
 
+    #[tokio::test]
+    async fn test_subschema_response() {
+        let mut ldap_handler = setup_bound_admin_handler(MockTestBackendHandler::new()).await;
+
+        let request = LdapSearchRequest {
+            base: "cn=Subschema".to_string(),
+            scope: LdapSearchScope::Base,
+            aliases: LdapDerefAliases::Never,
+            sizelimit: 0,
+            timelimit: 0,
+            typesonly: false,
+            filter: LdapFilter::Present("objectClass".to_string()),
+            attrs: vec!["supportedExtension".to_string()],
+        };
+
+        let current_time_utc = Utc::now().format("%Y%m%d%H%M%SZ").to_string().into_bytes();
+
+        let expected_response: Vec<LdapOp> = vec![
+            LdapOp::SearchResultEntry(LdapSearchResultEntry {
+                dn: "cn=Subschema".to_owned(),
+                attributes: vec![LdapPartialAttribute {
+                    atype: "structuralObjectClass".to_owned(),
+                    vals: vec![b"subentry".to_vec()]
+                },
+                LdapPartialAttribute {
+                    atype: "objectClass".to_owned(),
+                    vals: vec![b"top".to_vec(), b"subentry".to_vec(), b"subschema".to_vec(), b"extensibleObject".to_vec()]
+                },
+                LdapPartialAttribute {
+                    atype: "cn".to_owned(),
+                    vals: vec![b"Subschema".to_vec()]
+                },
+                LdapPartialAttribute {
+                    atype: "createTimestamp".to_owned(),
+                    vals: vec![current_time_utc.to_vec()]
+                },
+                LdapPartialAttribute {
+                    atype: "modifyTimestamp".to_owned(),
+                    vals: vec![current_time_utc.to_vec()]
```

Comment:
I hope no one is relying on that to do cache invalidation... Not actionable for now, I guess. Since we're lying either way, it's probably better to say that it always "changed" rather than always returning the epoch and have an old entry stay in a cache.